src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -94,24 +128,41 @@ interface PriceChartProps {
   timePeriod: TimePeriod
 }
 
-function formatDisplayPrice(value: number) {
-  const str = value.toFixed(9)
-  const [digits, decimals] = str.split('.')
-  // Displays longer string for numbers < $2 to show changes in both stablecoins & small values
-  if (digits === '0' || digits === '1')
-    return `$${digits + '.' + decimals.substring(0, 2) + decimals.substring(2).replace(/0+$/, '')}`
-
-  return formatUSDPrice(value)
-}
-
-function PriceChart({ width, height, prices, timePeriod }: PriceChartProps) {
+// eslint-disable-next-line import/no-unused-modules
```

Comment:
confirming this disable comment is actually necessary? I'm surprised there would ever be a need for it.